CMakeLists.txt

```diff
@@ -47,6 +47,11 @@ include (src/cmake/utilities.txt)
 # Build configurations.
 #--------------------------------------------------------------------------------------------------
 
+# Disable in source builds.
```

Comment:
It's not really a good idea. My main reason is that it generates a lot of files that we are not ignoring in the git config.
I prefer to disable it by default and if anyone needs to do in-source builds review the decision.